vars/alert.groovy

```diff
@@ -0,0 +1,99 @@
+import groovy.json.JsonSlurperClassic
+import groovy.json.JsonBuilder
+
+String disableAlerts(Map args=[:]) {
```

Comment:
We could add a bit of documentation what are args, and maybe example call of that function